thirdparty/fetch-thirdparty-deps.ps1

```diff
@@ -69,11 +69,11 @@ if (!(Test-Path "nunit.framework.dll"))
 
 if (!(Test-Path "windows/SDL2.dll"))
 {
-	echo "Fetching SDL2 from NuGet."
-	./nuget.exe install sdl2 -Version 2.0.3 -ExcludeVersion
-	cp sdl2.redist/build/native/bin/Win32/dynamic/SDL2.dll ./windows/
-	rmdir sdl2 -Recurse
-	rmdir sdl2.redist -Recurse
+	echo "Fetching SDL2 from libsdl.org"
+	$target = Join-Path $pwd.ToString() "SDL2-2.0.4-win32-x86.zip"
```

Comment:
Odd. You are correct.

When I tried this vs what I suggested earlier today this didn't work (it resulted in just the value of `$PWD`), but now it works just fine.

What is failing is:
```
File: "C:\projects\openra\thirdparty\download\windows\SDL2.dll" -> no files found.
Usage: File [/nonfatal] [/a] ([/r] [/x filespec [...]] filespec [...] |
   /oname=outfile one_file_only)
Error in script "packaging/windows/OpenRA.nsi" on line 112 -- aborting creation process
Command exited with code 1
```

Should you be using `7z e SDL2-2.0.4-win32-x86.zip -o .\windows SDL2.dll` instead of `7z e SDL2-2.0.4-win32-x86.zip -o.\windows SDL2.dll`?

The difference being a space after `-o`.